### DISCUSSION TITLE
Option to see compiled css in the editor

### DISCUSSION THREAD

**Discussion by pabli24** [openstyles/stylus#1489] [Ideas]
It can be pretty handy while using the preprocessors

example:
![stylus-output](https://user-images.githubusercontent.com/22481052/195432333-c09a0842-4e47-4d49-a181-246b694ad5f0.png)

**Comment by tophf** [comment]
How would that be useful? For example, I almost never need it except when debugging a particularly complicated syntactic construction, probably, in which case I open the online editor for less or stylus-lang.

**Reply by pabli24** [reply]
I also sometimes use the online editor so I thought it would be nice to have it built in. It would save me at least 5 seconds googling and copying it :D
Can be useful for beginners to learn how preprocessors and variables work or when they look through other people's code to understand what is going on. 

But I guess it's not worth implementing, while you can use the online editor or see the code in the DevTools.
